.github/workflows/CI.yml

```diff
@@ -184,3 +184,52 @@ jobs:
             which node
             node -v
             node src/lpython/tests/test_lpython.js
+
+  test_pip_pkgs:
```

Comment:
I would like to possible have a separate test case at the CI for testing pip installable packages (for example, similar to how we have unique CI jobs for different backends at LFortran). So, that if there is any failure, it is clear from the jobs which failed if the failure is in pip installed package tests or in regular tests.